server/service/src/invoice/inbound_shipment/update/generate.rs

```diff
@@ -370,6 +370,7 @@ pub fn generate_lines_and_stock_lines(
                 item_variant_id,
                 donor_id,
                 vvm_status_id: None,
+                campaign_id: None,
```

Comment:
I've kept the PR small for now, leaving updates to `campaign_id` to be handled by the relevant issues that need to manipulate its value. Unless we think otherwise - would be more than glad to add it to this PR.

The same applies to any other additions or edits involving `campaign_id`.